service/storage_service.cc

```diff
@@ -3711,6 +3715,25 @@ future<> storage_service::notify_cql_change(inet_address endpoint, bool ready) {
     }
 }
 
+bool storage_service::is_normal_state_handled_on_boot(gms::inet_address node) {
+    return normal_state_handled_on_boot.contains(node);
+}
+
+// Wait for normal state handler to finish on boot
+future<> storage_service::wait_for_normal_state_handled_on_boot(std::list<gms::inet_address> nodes, sstring ops, node_ops_id uuid) {
```

Comment:
Please log one message on INFO level at the beginning and end of this function.

Reply:
I thought it was too noise as info level, but this is a one time log during boot up. It is not bad as info level. Added in v2.